docs/products/cassandra/howto/disable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,117 @@
+
+* CCR enabled on a pair of Aiven for Apache Cassandra services
+
+Disable CCR in the console
+--------------------------
+
+In the console, use either of the two following methods to disable CCR on your service(s): split the services or delete one of them.
+
+Split the services
+''''''''''''''''''
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select the service for which you'd like to disable CCR.
+3. In the **Overview** tab of the service's page, navigate to **Cross Cluster Replication** and select **Split cluster**.
+
+   .. warning::
+
+      As soon as you split the cluster, the two services constituting the CCR pair become independent. It's not possible to recreate the CCR pair with the same service as a replica. You can select any other Aiven for Cassandra service for that purpose.
+   
+4. When in the **Warning** popup, make sure you understand the impact and select **Split cluster**.
+
+.. topic:: Result
+
+   You service no longer has a cross-cluster replica.
+
+Delete a service
+''''''''''''''''
+
+To disable CCR on your service, delete the replica service that is connected to your services for CCR purposes.
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select an Aiven for Apache Cassandra service on which you'd like to enable CCR.
+3. In the **Overview** tab of the service's page, navigate to the **Cross Cluster Replication** section and select the name of the replica service provided in the CCR description, which is supposed to take you to the replica service's page.
+4. In the **Overview** tab of the replica service, select **Delete service** from the top right corner.
+
+   .. warning::
+
+      As soon as you delete the replica service, CCR gets disabled for your service that is now connected to the replica service for CCR purposes.
+
+5. When in the **Delete confirmation** popup, make sure you understand the impact, copy-paste the service name, and select **Delete**.
+
+.. topic:: Result
+
+   You've disable CCR on your service, which no longer has a cross-cluster replica since the replica service has been deleted.
+
+Disable CCR with CLI
+--------------------
+
+You can disable CCR for your Aiven for Apache Cassandra service using the Aiven CLI to delete one of the services constituting the CCR pair.
+
+.. tip::
+
+   Check out how to get started with the Aiven CLI in :doc:`Aiven CLI </docs/tools/cli>`.
+
+Use the :ref:`avn service terminate <avn-cli-service-terminate>` command to disable CCR on your service by deleting its replica service.
+
+.. code-block:: bash
+
+   avn service terminate replica_service_name
+
+Disable CCR with API
+--------------------
+
+You can disable CCR for your Aiven for Apache Cassandra service(s) by calling the `terminate a service <https://api.aiven.io/doc/#tag/Service/operation/ServiceDelete>`_ endpoint to delete one of the services that constitute the CCR pair.
```

Comment:
is it called "terminate endpoint" anywhere? it is ServiceDelete, maybe the "delete service endpoint"?